app/controllers/community_memberships_controller.rb

```diff
@@ -30,7 +30,7 @@ def pending_consent
 
   def give_consent
     form_params = params[:form] || {}
-    values = Form.call(form_params)
+    values = Form.call(form_params.to_unsafe_hash)
```

Comment:
Would it be too complex to start whitelisting parameters and avoid using unsafe hashes, @aivils? Here and all other places `.to_unsafe_hash` is called.

Reply:
@bladealslayer 
Avoiding unsafe hashes would be breaking changes. There is a warning https://github.com/sharetribe/sharetribe/blob/master/app/utils/entity_utils.rb#L416

Removing warning and refactoring EntityUtils is a serious step to affect the whole project.